exercises/diffie-hellman/diffie-hellman.spec.js

```diff
@@ -0,0 +1,73 @@
+var DiffieHellman = require('./diffie-hellman');
+
+describe('diffie-hellman', () => {
```

Comment:
Since we're targeting ES5 in this track, it would be best if you used traditional functions instead of arrow functions.